nfs/inc/rpc_task.h

```diff
@@ -636,6 +636,22 @@ struct setattr_rpc_task
     int to_set;
 };
 
+struct statfs_rpc_task
+{
+    fuse_ino_t get_ino() const
+    {
+        return ino;
+    }
+
+    void set_ino(fuse_ino_t inode)
```

Comment:
we should use ino for inode number variable (of type fuse_ino_t) and inode for nfs_inode, so arg should be _ino